hack/windows/Install-OVS.ps1

```diff
@@ -187,33 +129,268 @@ function InstallOVS() {
             }
         }
     }
+}
 
-    # Install OVS kernel driver.
-    Log "Installing OVS kernel driver"
-    $VMMSStatus = $(Get-Service vmms -ErrorAction SilentlyContinue).Status
-    if (!$VMMSStatus) {
-        $VMMSStatus = "not exist"
+function CheckAndInstallVCRedists {
```

Comment:
Just to confirm: if someone runs `.\Install-OVS.ps1 -InstallUserspace $true` from the Windows host (which is still supported, as per the documentation, for the native OVS services case), will this function be a no-op? I ask because `redist` is not included in https://downloads.antrea.io/ovs/ovs-3.0.5-antrea.1-win64.zip, but only in the container image.

Reply:
Good catch, I would update here like this, if the vcredist files are not installed on the host, then check if the local redist directory is provided, if yes, install with the local files; otherwise download and install the files.